src/mozilla_sec_eia/models/sec10k/utils/cloud.py

```diff
@@ -278,11 +196,12 @@ def get_filings(
         """
         filings = []
         for _, filing in filing_selection.iterrows():
-            blob = self.get_filing_blob(filing["year_quarter"], filing["filename"])
             local_path = self.get_local_filename(cache_directory, filing)
-            filing_path = self.cache_blob(blob, local_path)
+            if not local_path.exists():
+                with local_path.open("w") as f:
+                    f.write((self.filings_bucket_path / filing.filename).read_text())
```

Comment:
Flagging that without `self.get_filing_blob` to get the year quarter, this path (`self.filings_bucket_path / filing.filename`) no longer works to get the path to the filing in GCS. You could get the year quarter from the metadata instead?

Reply:
I fixed this by updating to:

```
year_quarter = filing["year_quarter"]
if not local_path.exists():
    with local_path.open("w") as f:
        f.write(
            (
                self.filings_bucket_path
                / "sec10k"
                / f"sec10k-{year_quarter}"
                / filing.filename
            ).read_text()
        )
```
(didn't want to commit directly to your branch though in case you wanted to make more changes here)